### DISCUSSION TITLE
resize image respect to its original size

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#5670] [Q&A] [answered]
Thanks for the great job. I want to resize my image to the half of original size using MONAI transform. But dont find an appropriate transform. I found monai.transform.Resize can resize an image to a given size but can not to a size relevent to orininal image size.

**Comment by KumoLiu** [comment, marked as answer]
Hi @YingdiZhang, specifying `spatial_size` to be half the size of the original image doesn't work?

**Reply by YingdiZhang** [reply]
Thanks for the reply, I didnt express clearly. My 3D CT images have different size, which is range from [290, 290, 203] to [816, 816, 476]. While monai.transform.Resize can not resize a image directly with fraction of original size

**Reply by wyli** [reply]
I think you can use `Zoom` 

https://github.com/Project-MONAI/MONAI/blob/78d4f42062a7e22c2be6e947bb6e7481ef89df25/monai/transforms/spatial/array.py#L1128-L1137

**Reply by YingdiZhang** [reply]
Thanks for your answer, it solved my problem
